megatron/global_vars.py

```diff
@@ -152,6 +160,45 @@ def _set_tensorboard_writer(args):
                   'no TensorBoard logs will be written.', flush=True)
 
 
+def _set_wandb_writer(args):
+    """Set wandb writer."""
+    global _GLOBAL_WANDB_WRITER
+    _ensure_var_is_not_initialized(_GLOBAL_WANDB_WRITER,
+                                   'wandb writer')
+    getattr(args, 'wandb_project', '')
+    getattr(args, 'wandb_exp_name', '')
+
+    if args.rank == (args.world_size - 1):
```

Comment:
For consistency, why not?

```suggestion
    if is_last_rank:
```

Reply:
I try this but encouter error: ImportError: cannot import name 'is_last_rank' from partially initialized module 'megatron' (most likely due to a circular import). I think till now it is good to retain this part of code.